### PR TITLE
WIP: V4R update

### PR DESCRIPTION
**Pull request by ToMadoRe**
associated update to https://github.com/strands-project/v4r/pull/73

### CODE REVIEW

File: temporal_filtering/package.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  <!-- Use run_depend for packages you need at runtime: -->
+  <!--   <run_depend>message_runtime</run_depend> -->
+  <!-- Use test_depend for packages you need only for testing: -->
+  <!--   <test_depend>gtest</test_depend> -->
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>roscpp</build_depend>
+  <build_depend>rospy</build_depend>
+  <build_depend>std_msgs</build_depend>
+  <build_depend>std_srvs</build_depend>
+  <build_depend>opencv</build_depend>
```
Comment (marc-hanheide) [inline temporal_filtering/package.xml]: This is not a valid rosdep key. This is what failed in https://lcas.lincoln.ac.uk/jenkins/job/pr-indigo-v4r_ros_wrappers/49/ You may want to use `cv_bridge` instead. Also below in `<run_depend>`

**Review by marc-hanheide — CHANGES_REQUESTED** [review]
the dependencies need to be fixed

File: temporal_filtering/package.xml (hunk 1 above)
Comment (marc-hanheide) [reply temporal_filtering/package.xml]: @ToMadoRe I think you ought to be fixing this.

**Review by marc-hanheide — APPROVED** [review]

**Comment by ToMadoRe** [comment]
retest this please

**Pull request merged**